### PR TITLE
Update Slack-channel-usage page for new slack space.

### PR DESCRIPTION
**Pull request by edennihy**

### CODE REVIEW

**Review by frossie — APPROVED** [review]
Looks good thanks

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 1]
```diff
@@ -127,40 +148,15 @@ For items having substantial impact, scheduling should first be carried out as p
   
 - This channel is specific to summit related activities.
   Announcements for testing using other test-stands are to be made on test-stand specific channels, following the same rules.
-  This includes #rubinobs-tucson-teststand and #rubinobs-base-teststand
```
Comment (isotuela) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:130]: . at the end of the sentence.

File: Daytime-Nighttime-Interactions/slack-channel-usage.rst [hunk 2]
```diff
@@ -127,40 +148,15 @@ For items having substantial impact, scheduling should first be carried out as p
...
-
-.. figure:: ./_static/dialog_example.png
-    :name: #summit-control-room example
-    :width: 560
-    :height: 399
-
-    An example of the type of troubleshooting conversation that should take place in #summit-control-room.
+  This includes #tucson-teststand and #base-teststand
 
+- The summit is an active observatory with many teams working on different tasks. Personnel on the summit should never be discouraged from announcing their work as impacts may be unforeseen or unpredictable.
```
Comment (isotuela) [inline Daytime-Nighttime-Interactions/slack-channel-usage.rst:153]: One sentence per line here would help with future diff.

**Review by isotuela — APPROVED** [review]
Thank you Erik for these updates. 

I have added few very minor comments; feel free to implement them at your discretion. 

Cheers, 
Ioana

**Pull request merged**